sdk/core/azure-core-experimental/azure/core/experimental/transport/_pyodide.py

```diff
@@ -36,10 +36,10 @@
 from azure.core.utils import CaseInsensitiveDict
 
 from azure.core.rest._http_response_impl_async import AsyncHttpResponseImpl
-from azure.core.pipeline.transport import (
+from azure.core.pipeline.transport import (  # pylint: disable=non-abstract-transport-import, no-name-in-module
```

Comment:
black changed location of the disable line --adding fix here to allow core to pass